pebblo/app/service/loader/loader_doc_service.py

```diff
@@ -290,6 +293,23 @@ def process_request(self, data):
             )
             self.db.update_data(loader_obj, app_loader_details)
 
+            # Creating loader response
+            loader_details = self.data.get("loader_details", {})
+            docs = self.data.get("docs", [])
+            loader_response_output = []
+            for doc in docs:
+                doc_obj = LoaderDocs(
+                    pb_id=doc["pb_id"],
```

Comment:
Is it possible to use pydantic approach in place of assigning every variable ?

Reply:
yes, we can do that, but as of now we are using selected fields from `doc` and hence preparing it here.